packages/blocks/shuffle/src/shuffle.tsx

```diff
@@ -0,0 +1,153 @@
+import React, { useEffect, useRef, useState } from "react";
+import {
+  BlockComponent,
+  useGraphBlockService,
+} from "@blockprotocol/graph/react";
+// eslint-disable-next-line no-restricted-imports
+import Button from "@mui/material/Button";
```

Comment:
Why *are* we using a restricted import here? Maybe there should be a comment explaining why?

Reply:
It was being used because the block's eslint config extends the root project one on which we only allow using our custom wrapper Button component. I removed this line and instead created an override for this rule in the block's `.eslintrc.js` (19ab51bd61bc009afebcf112dd2920ee0c09f694)